salt/flink/templates/flink-service.service.tpl

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Flink History-Server Service
+
+[Service]
+Type=forking
+WorkingDirectory={{ install_dir }}
+ExecStart={{ install_dir }}/bin/historyserver.sh start
+Restart=always
+RestartSec=2
+
```

Comment:
"/bin/historyserver.sh stop" is missing.

Reply:
added.